modules/Spots/controller.php

```diff
@@ -548,12 +646,14 @@ public function action_getActivitiesSpotsData()
     public function action_createActivitiesSpotsData($filepath)
     {
         global $mod_strings;
+        global $app_list_strings;
+
         $returnArray = [];
         $db = DBManagerFactory::getInstance();
 
         $mysqlQueryCalls = <<<EOF
         SELECT
-            'call' as type
+            'LBL_ACTIVITIES_CALL' as type
```

Comment:
The column name in database queries shouldn't be translated with a label.  It's code, not seen by the user in the browser.  More below, same comment.  Labels are LBL_ACTIVITIES_CALL , LBL_ACTIVITIES_MEETING , LBL_ACTIVITIES_TASK.